examples/static_graph_optimizations/ptb/train_ptb_custom_loop.py

```diff
@@ -0,0 +1,304 @@
+"""Sample script of recurrent neural network language model.
```

Comment:
Add `with static subgraph optimization` to make clear how this is different from the other one.